datalad/distribution/tests/test_install.py

```diff
@@ -208,6 +208,25 @@ def test_install_simple_local(src, path):
         # no content was installed:
         ok_(not ds.repo.file_has_content('test-annex.dat'))
 
+    # installing it again, shouldn't matter:
+    with swallow_logs(new_level=logging.INFO) as cml1:
+        ds = install(path=path, source=src)
+        assert_in("{0} appears to be installed already.".format(ds), cml1.out)
+        ok_(ds.is_installed())
+
+    # neither should installing just the installed one (without source):
+    with swallow_logs(new_level=logging.INFO) as cml2:
+        ds = install(path=path)
+        ok_(False, "DEBUG:\n%s" % cml2.out)
```

Comment:
what are you trying to check here?  Note that for asserting within logs, you can do now

```python
cml2.assert_logged("whatever you expect msg to be", level="DEBUG")
```
find/see examples in the codebase